modules/ztp-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-prep_{context}"]
+= Preparing the {sno} clusters for the image-based upgrade
+
+When you deploy the {lcao} on a cluster, the Operator creates an `imageBasedUpgrade` CR.
+You use this CR throughout the upgrade process to move through the different stages.
+
+In the `Prep` stage, the {lcao} ensures that the nodes are ready for the upgrade by checking if the cluster meets certain conditions.
+You can complete this stage well before starting the upgrade.
+
+After you start the `Prep` stage by applying the `imageBasedUpgrade` CR with {cgu-operator}, {cgu-operator} applies a policy that starts the following procedures:
+
+* Installing and configuring the OADP Operator
+* Adding a `DataProtectionApplication` CR that points to the S3 backend
+* Adding a secret for the S3 credentials that are referenced in the `DataProtectionApplication` CR
+
+These resources are deleted after the upgrade is complete or if a rollback is initiated.
+
+.Prerequisites
+
+* You have installed {cgu-operator}.
+* You have installed {lcao}.
+* You have generated a seed image from a compatible seed cluster.
+
+// are there other prereqs?
+
+.Procedure
```

Comment:
This has to be done as part of the installation of the target system

Reply:
Correct, also include installation of the OADP operator as prereq, otherwise the DataProtectionApplication CR that is stated to be applied by TALM a couple of lines above would not be applied since cluster does not understand that CR.